src/components/MovieCard/MovieCard.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './MovieCard.scss';
+
+const MovieCard = ({ episode, title, releaseDate }) => {
+  return (
+  <article className='movie-card'>
+    <h2>Episode {episode}</h2>
+    <h3>{title}</h3>
+    <p>{releaseDate}</p>
+    <button>View Characters</button>
+  </article>
+  )
+};
+
+export default MovieCard;
```

Comment:
Another next-steps thing for us to think through would be where we will store/pass the character data.  Will this be stored in App and passed through the DisplayContainer > MovieCard > CharacterCard?   

If we set it up with a selectedMovie state in App, we could conditionally pass only the character props that are needed for that movie.  (If no movie selected, pass all or none.  Once a movie is selected, update the state of App to reflect this then have it conditionally re-render passing the appropriate Character props.)